src/server/admin/server/admin_test.go

```diff
@@ -845,6 +836,90 @@ func TestMigrateFrom1_7(t *testing.T) {
 	require.Equal(t, 6, len(commits))
 }
 
+func TestMigrateAuthFrom1_11(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	// Clear pachyderm cluster (so that next cluster starts up in a clean environment)
+	tu.DeleteAll(t)
+	defer tu.DeleteAll(t)
+
+	c := tu.GetPachClient(t)
+	c.SetAuthToken("known-auth-token")
+	tu.UpdateAuthToken(tu.AdminUser, "known-auth-token")
+
+	// Restore dumped metadata (now that objects are present)
+	md, err := os.Open(path.Join(os.Getenv("GOPATH"),
+		"src/github.com/pachyderm/pachyderm/etc/testing/migration/v1_11/auth.metadata"))
+	require.NoError(t, err)
+	require.NoError(t, c.RestoreReader(snappy.NewReader(md)))
+	require.NoError(t, md.Close())
+
+	// Wait for final imported commit to be processed
+	commitIter, err := c.FlushCommit([]*pfs.Commit{client.NewCommit("left", "master")}, nil)
+	require.NoError(t, err)
+	commitInfos := collectCommitInfos(t, commitIter)
```

Comment:
```suggestion
	commitInfos, err := c.FlushCommitAll([]*pfs.Commit{client.NewCommit("left", "master")}, nil)
	require.NoError(t, err)
```

Tiny bit cleaner to use helper function `FlushCommitAll`